### DISCUSSION TITLE
Should we have a default catchall/fallback job handler?

### DISCUSSION THREAD

**Discussion by wanghanlin** [spatie/laravel-stripe-webhooks#127] [Ideas]
In stripe cli for local webhook development, stripe will proxy and forward all event types, which get write into database without a function to process it.
Some time, we could enable more event types than needed on stripe console as well.
Should we have a default job handler built-in or open for integration that do nothing but mark those extra webhook calls that write into DB as success

Suggestion:
https://github.com/spatie/laravel-stripe-webhooks/blob/main/src/ProcessStripeWebhookJob.php#L35
change the default  `''` to a new Job class built-in package, or have an option in config `stripe-webhooks.jobs.fallback` or something like `stripe-webhooks.default_jobs` for developer to code for a fallback handler

**Comment by freekmurze** [comment]
Feel free to create a PR that adds this functionality. Make sure to update the readme and tests.

**Reply by wanghanlin** [reply]
Thanks, PR #128 is created, please let me know if there is any change needed.
